src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskExecuteReconfigureControllerTest.java

```diff
@@ -1029,9 +1038,13 @@ public void should_reconfigure_camunda_task_attributes_after_validation_reconfig
                     .body("task.reconfigure_request_time", nullValue())
                     .body("task.last_reconfiguration_time", notNullValue())
                     .body("task.task_title",
-                          is("name - reconfigTaskAttributesTask - state - ASSIGNED - category - Protection"))
+                          is("name - " + taskName + " - state - ASSIGNED - category - Protection"))
                     .body("task.due_date", notNullValue())
-                    .body("task.role_category", is("CTSC"));
+                    .body("task.role_category", is("CTSC"))
+                    .body("task.additional_properties", equalToObject(Map.of(
+                        "key1", "value1",
+                        "roleAssignmentId", roleAssignmentId
```

Comment:
Just to check ... the process categories should only be available in the DMN during the Initiation and not the Reconfiguration.  I think in the PRL scenario we'd normally set the canReconfigure flag to false.  Not sure if that's happening here?